source/components/atoms/Stepper/Stepper.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { View } from 'react-native';
```

Comment:
Remove View as it is not used

Reply:
![Simulator Screen Shot - iPhone 11 - 2020-06-02 at 11 42 32](https://user-images.githubusercontent.com/26883320/83505639-5aad2d00-a4c6-11ea-86d2-81a15f20330e.png)